3rdparty/libgcov/LICENSE.TXT

```diff
@@ -0,0 +1,91 @@
+==============================================================================
+compiler_rt License
```

Comment:
Can this be added as a submodule instead of forking the code?
I would really like us to be using submodules for all additions so we don't keep adding to the debt we already have.

Reply:
Sure. This PR is still considered as a draft and mainly for testing the CI integration. I will address the comments in the next iteration.